doc/tools/stlink-v21.rst

```diff
@@ -59,15 +59,14 @@ Host Tools and Firmware
 Download and install the `Segger J-Link Software and Documentation Pack`_ to
 get the J-Link GDB server for your host computer.
 
+Open new terminal and export ``OPENOCD_INTERFACE=jlink`` environment variable
```

Comment:
Change to:

    To make OpenOCD use the J-Link tools, open a new terminal and use::

        export OPENOCD_INTERFACE=jlink

    before flashing and debugging.